src/wrappers/EventsListener.tsx

```diff
@@ -1,14 +1,20 @@
 import { atom, useAtomValue, useSetAtom } from 'jotai';
 
+import { activePaneAtom, closeEditorFromPaneAtom } from '../atoms/editorActions';
 import { activePaneContentAtom } from '../atoms/paneActions';
-import { RefStudioEvents } from '../events';
+import { PaneEditorId } from '../atoms/types/PaneGroup';
+import { emitEvent, RefStudioEvents } from '../events';
 import { useListenEvent } from '../hooks/useListenEvent';
-import { asyncNoop } from '../lib/noop';
+import { asyncNoop, noop } from '../lib/noop';
 
 export function EventsListener({ children }: { children?: React.ReactNode }) {
   const saveActiveFile = useSaveActiveFile();
+  const closeActiveEditor = useCloseActiveEditor();
+  const closeEditor = useCloseEditor();
 
   useListenEvent(RefStudioEvents.menu.file.save, saveActiveFile);
+  useListenEvent(RefStudioEvents.menu.file.close, closeActiveEditor);
+  useListenEvent(RefStudioEvents.editors.close, closeEditor);
```

Comment:
wondering if we can merge these two with an optional payload for the `menu.file.close`